monkey/monkey_island/cc/next_ui/src/redux/features/api/authentication/authenticationActions.tsx

```diff
@@ -0,0 +1,3 @@
+export const AuthenticationActions = {
+    logout: { type: 'LOGOUT' }
```

Comment:
can't we just call to dispatch the logout (same as the setTimer for example)?

Reply:
We are calling it (same as the setTimer). Except `setTimer` is defined in a slice, this action is defined standalone, since there are no reducers for it